.travis.yml

```diff
@@ -45,14 +47,13 @@ branches:
     - master
     - /^test-.*$/
 
-# container-based infrastructure
-sudo: false
+sudo: required
```

Comment:
It allows us to run the le-auto integration tests, which need sudo almost by definition, on Travis. (I isolate them in a Docker container so our scribbling doesn't interfere with other tests and so they can be run locally.)